scripts/create-agent.ts

```diff
@@ -48,6 +48,8 @@ class VargasJRAgentCreator {
       
       await this.setupInstance(instanceDetails, keyPairName);
       
+      await this.waitForInstanceHealthy(instanceId);
```

Comment:
try catch this for now, where in the case of error, we print the error and don't exit the action yet